prow/plugins/lifecycle/close_test.go

```diff
@@ -169,6 +183,36 @@ func TestCloseComment(t *testing.T) {
 			shouldClose:   false,
 			shouldComment: true,
 		},
+		{
+			name:          "close by author as not planned",
+			action:        github.GenericCommentActionCreated,
+			state:         "open",
+			stateReason:   "not_planned",
+			body:          "/close not-planned",
+			commenter:     "author",
+			shouldClose:   true,
+			shouldComment: true,
+		},
+		{
+			name:          "close by author as not planned, trailing space.",
+			action:        github.GenericCommentActionCreated,
+			state:         "open",
+			stateReason:   "not_planned",
+			body:          "/close not-planned \r",
+			commenter:     "author",
+			shouldClose:   true,
+			shouldComment: true,
+		},
+		{
+			name:          "close by author, multiple spaces in between.",
```

Comment:
For completeness, can we also add test cases with comment by a non-collaborator?